wazo_call_logd/plugins/cdr/http.py

```diff
@@ -65,6 +65,7 @@
     'source_user_uuid',
     'source_line_id',
     'tags',
+    'remote_party',
```

Comment:
I don't see how `remote_party` could be inserted in a CSV format? I think we should not have `remote_party` at all in the CSV.

Reply:
Fixed it